lm_eval/base.py

```diff
@@ -713,9 +719,11 @@ def fewshot_context(
                 fewshotex = [x for x in fewshotex if x != doc][:num_fewshot]
 
             labeled_examples = (
-                "\n\n".join(
+                "\n\n" +
+                "".join(
                     [
-                        self.doc_to_text(doc) + self.doc_to_target(doc)
+                        self.doc_to_text(doc) + template_tags[template_type][0] + \
+                        self.doc_to_target(doc) + template_tags[template_type][1]
```

Comment:
I'm not fully sure, but looks like current state is not equivalent to old one for case `template_type == none`, possibly  `template_tags[template_type][1]` should be `\n\n` instead of empty string